### PR TITLE
Remove TX Status limits, Bump Ethers, Fix formatting errors for some networks

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

**Comment by 409H** [comment]
See CH7422

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "date-fns": "2.16.1",
     "ethereumjs-abi": "0.6.8",
     "ethereumjs-util": "6.0.0",
-    "ethers": "4.0.48",
+    "ethers": "5.0.26",
```
Comment (Mrtenz) [inline package.json:41]: Maybe explicitly install `@ethersproject/*` stuff rather than `ethers`.

File: src/helpers/transaction.ts [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
-  parseTransaction,
+  parse as parseTransaction,
   recoverAddress,
-  serializeTransaction,
+  serialize as serializeTransaction,
   Transaction
-} from 'ethers/utils';
+} from '@ethersproject/transactions';
+import { formatEther } from '@ethersproject/units';
+import { addHexPrefix, toChecksumAddress } from 'ethereumjs-util';
```
Comment (Mrtenz) [inline src/helpers/transaction.ts:11]: We can use ethers.js here instead of ethereumjs.

File: src/services/EthService/utils/providerWrappers.ts [hunk 3]
```diff
@@ -37,7 +37,8 @@ export const getTransactionReceiptFromHash = async (
   await provider
     .getTransactionReceipt(txHash)
     .then((receipt) => receipt)
-    .catch(() => {
+    .catch((err) => {
+      console.error(err);
```
Comment (Mrtenz) [inline src/services/EthService/utils/providerWrappers.ts:41]: Did you forget to remove this?

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: src/helpers/transaction.ts (hunk 2 above)
Comment (FrederikBolding) [reply src/helpers/transaction.ts:11]: What are these functions called in `ethers.js`?

File: package.json (hunk 1 above)
Comment (FrederikBolding) [reply package.json:41]: Yeah, just don't know which ones we need at the moment 😄

File: src/helpers/transaction.ts (hunk 2 above)
Comment (Mrtenz) [reply src/helpers/transaction.ts:11]: * [`getAddress`](https://github.com/ethers-io/ethers.js/blob/master/packages/address/src.ts/index.ts#L77) in `@ethersproject/address`
* Couldn't find a proper replacement for `addHexPrefix` but if we want to remove `ethereumjs-util` fully at some point, it should be easy enough to make a util function for that.

File: package.json (hunk 1 above)
Comment (Mrtenz) [reply package.json:41]: As of now:
* `@ethersproject/bignumber`
* `@ethersproject/address`
* `@ethersproject/providers`
* `@ethersproject/transactions`
* `@ethersproject/bytes`
* `@ethersproject/abstract-provider`
* `@ethersproject/units`
* `@ethersproject/wallet`

**Review by Mrtenz — DISMISSED** [review]

**Comment by FrederikBolding** [comment]
Thanks for the review @Mrtenz 

I think @bowensanders should give this a look over before merging though!

**Comment by bowensanders** [comment]
I have just tested all available networks via tx's from their respective block explorers. 

I say this is OK to GO

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**